### ISSUE TITLE
[accessibility] "more" (three-dot) button unreachable with Voiceover on iPhones

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/ios-app#691]
With Voiceover enabled, it's no longer possible to reach the "more" (three-dot) button in the file list on iPhones (left side). On iPad, it's still working (right side).

![voiceover](https://user-images.githubusercontent.com/214010/80605328-3c6d9080-8a33-11ea-8d78-635727ac7206.gif)

iPhone 11 Pro Max on the left, iPad Pro 12.9 2018 on the right, both with 13.41.

Tested with TestFlight 1.3.2 (158)
app: f5d5ff96, sdk: 629f413

**Comment by hosy** [comment]
I can not reproduce it on my iPhone. 
![image](https://user-images.githubusercontent.com/736109/80627575-77ca8800-8a50-11ea-9401-fc4e463d597b.gif)

**Comment by hosy** [comment]
@michaelstingl I tested it without a pointer device only with my finger. You used a connected trackpad?

**Comment by michaelstingl** [comment]
I reproduced it also without pointer etc. Could you try with more files in the folder to bring up the index bar on the right?

**Comment by hosy** [comment]
I also tried it with index bar and it works. Otherwise I cannot find a problem in the implementation. @jesmrec can you check it too?

**Comment by michaelstingl** [comment]
Maybe try the bigger iPhone in Simulator?

**Comment by hosy** [comment]
Accessibility does not work in Simulator. Furthermore Accessibility inspector is broken since Catalina 🙄

**Comment by hosy** [comment]
For me it looks like an iOS bug.

**Comment by michaelstingl** [comment]
> For me it looks like an iOS bug.

Thanks for testing. I adjusted priorities accordingly.

**Comment by michaelstingl** [comment]
Unable to reproduce on iPhone 8

**Comment by jesmrec** [comment]
I tested this at the time of https://github.com/owncloud/ios-app/pull/701 and it is not reproducible on my side. three-dot-button to open the card is reachable with VoiceOver
